app/models/movie.rb

```diff
@@ -28,10 +28,14 @@ class Movie < ApplicationRecord
   mount_uploader :background, BackgroundUploader
 
   delegate :genre_name, to: :genre
-  default_scope { where(status: 1) }
+  default_scope{where(status: 1)}
   scope :by_title, ->(title){where("title LIKE ?", "%#{title}%")}
+  scope :by_director, ->(director){where("director LIKE ?", "%#{director}%")}
+  scope :by_status, ->(status){where("status LIKE ?", "%#{status}%")}
```

Comment:
```suggestion
  scope :by_status, ->(status){where(status: status)}
```
chứ e
lạm dụng like quá

Reply:
Vâng ạ !